spec/acceptance/ntp_daemon_opts_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper_acceptance'
+
+config = if os[:family] == 'redhat'
+           '/etc/sysconfig/ntp'
+         else
+           '/etc/default/ntp'
+         end
+
+describe 'ntp class with daemon options:', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+  it 'run module twice idempotently' do
+    idempotent_apply(default, pp, {})
```

Comment:
Not sure how I didn't notice this before, but where is the manifest you are applying?

Reply:
It seems to have disappeared